server/src/test/java/com/objectcomputing/checkins/services/TestContainersSuite.java

```diff
@@ -89,32 +72,6 @@ public void setup() {
         flyway.migrate();
     }
 
-    @Override
-    public Map<String, String> getProperties() {
-        HashMap<String, String> properties = new HashMap<>();
-        properties.put("datasources.default.url", getJdbcUrl());
-        properties.put("datasources.default.username", getUsername());
-        properties.put("datasources.default.password", getPassword());
-        properties.put("datasources.default.dialect", "POSTGRES");
-        properties.put("datasources.default.driverClassName", "org.testcontainers.jdbc.ContainerDatabaseDriver");
-        properties.put("flyway.datasources.default.clean-schema", "true"); // Needed to run Flyway.clean()
-        properties.put("mail-jet.from_address", "someEmail@gmail.com");
```

Comment:
Not sure if these mail jet configs were actually needed here, and if so may need to be migrated into the test yml?

Reply:
I don't think this method was ever called as none of the tests that extend this are `@TestInstance(TestInstance.Lifecycle.PER_CLASS)`